modules/analysis/src/endpoints.rs

```diff
@@ -28,35 +33,25 @@ pub fn configure(config: &mut web::ServiceConfig, db: Database) {
         search_component_deps,
         get_component_deps,
     ),
-    components(schemas(
-        crate::model::PackageNode,
-        crate::model::AnalysisStatus,
-        crate::model::AncestorSummary,
-        crate::model::AncNode,
-        crate::model::DepSummary,
-        crate::model::DepNode,
-    )),
     tags()
 )]
 pub struct ApiDoc;
 
 #[utoipa::path(
-    context_path = "/api",
     tag = "analysis",
     operation_id = "status",
     responses(
         (status = 200, description = "Analysis status.", body = AnalysisStatus),
     ),
 )]
-#[get("/v1/analysis/status")]
+#[get("/status")]
```

Comment:
Personally, I like seeing the components of the path most relevant to the fn right next to the fn. In this case, I'd prefer `#[get("/analysis/status")]`, I think. I could live with the `/v1` in there, too, but we could defer that until we have a `/v2`.

My only reason is that it makes it easy to construct the URL for curl, etc rather than having to look up the `CONTEXT_PATH` const, so it's a minor nit if you feel strongly otherwise.

Reply:
It made live a bit easier before the most recent change (of dropping the manual collection altogether). Now that we no longer need to hand-craft the OpenAPI struct, I can try to revert this. Though I am not sure how this affect the spec generation. But I'll find out.

Reply:
Ok, I reverted that part. I think it should work just fine now.